.gitignore

```diff
@@ -21,6 +21,9 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-lock.json
+package.json
```

Comment:
Please do not exclute them in the gitignore :)

Reply:
Ok I will make the changes again then😅

Reply:
I have done all requested now can you have a  look🙂